### PR TITLE
Variables and forms for setting standalone battery timestep

### PR DESCRIPTION
**Pull request by brtietz**
Add new page for stand alone battery timestep, and add it to the "battery" group before the cell and system info.

List numbers don't seem to translate directly into ssc, so add some lk to deal with this.

Mismatches with load, revenue, and curtailment lengths are enforced within ssc as appropriate, the other battery models don't have red text or pop up warnings for these, so I haven't added them here.

Does this require changes to the version upgrade script, or is that a separate effort? 

Goes with https://github.com/NREL/ssc/pull/1227

### CODE REVIEW

**Comment by mjprilliman** [comment]
It looks like the 'Battery Standalone Timestep.json' UI form didn't make it into the commit here.

**Review by mjprilliman — COMMENTED** [review]
This error message could use some additional text as there is no weather file (BTM standalone battery mismatch between timestep and load timestep). 

image

On the UI form, fit the group box to the contents, give a larger buffer between the text and the group box for different fonts on different platforms, and reduce the width of the dropdown some. 

On version upgrade, maybe set timestep_minutes and timestep_minutes_ui to the corresponding value from curtailment, revenue, or load length from the old case?

**Review by mjprilliman — APPROVED** [review]
Looks good, just a couple nitpicks and some things to check when going through the release checklist.

File: deploy/runtime/ui/Battery Standalone Timestep.json [hunk 1]
```diff
@@ -0,0 +1,184 @@
+{
```
[the remark this answers is not in the document]
Comment (cpaulgilman) [reply deploy/runtime/ui/Battery Standalone Timestep.json:1]: I redid this a bit, so it should work across platforms:

image

**Comment by cpaulgilman** [comment]
@brtietz, @mjprilliman Any objections to moving this form to the Battery Cell and System page so that it appears under the "Chemistry" group? Right now when you create a new Standalone Battery case, all you see is the simulation time step input with the Battery item in the navigation collapses, which might cause some folks to panic.

**Comment by brtietz** [comment]
> @brtietz, @mjprilliman Any objections to moving this form to the Battery Cell and System page so that it appears under the "Chemistry" group? Right now when you create a new Standalone Battery case, all you see is the simulation time step input with the Battery item in the navigation collapses, which might cause some folks to panic.

No objections to moving it. I trust your intuition on user expectations & panic levels.

**Comment by cpaulgilman** [comment]
> > @brtietz, @mjprilliman Any objections to moving this form to the Battery Cell and System page so that it appears under the "Chemistry" group? Right now when you create a new Standalone Battery case, all you see is the simulation time step input with the Battery item in the navigation collapsed, which might cause some folks to panic.
> 
> No objections to moving it. I trust your intuition on user expectations & panic levels.

I moved it below Battery Cell and System as a compromise -- simpler implementation than putting it on Battery Cell and System page and avoids the potential panic.

image

**Review by cpaulgilman — APPROVED** [review]

**Pull request merged**